components/proxy/src/main/java/com/hotels/styx/StyxServer.java

```diff
@@ -117,7 +117,7 @@ private static StyxServer createStyxServer(String[] args) {
         LOG.info("Styx logConfigLocation={}", startupConfig.logConfigLocation());
 
         StyxServerComponents components = new StyxServerComponents.Builder()
-                .registry(Metrics.globalRegistry)
+                .registry(new MicrometerRegistry(new CompositeMeterRegistry()))
```

Comment:
could we just have a new PrometheusMeterRegistry here?

Reply:
i.e. Do we need a CompositeMeterRegistry if there is only in Registry in it?

Reply:
I am not certain about this part. I can look into how the `PrometheusMeterRegistry` is added. It probably needs config injection to construct.

Reply:
I just checked - it's created like this: `new PrometheusMeterRegistry(PrometheusConfig.DEFAULT)`

Reply:
It seems to be added to the parent registry in the constructor of one of the admin interface handlers. This is definitely not the right place. I will move it.

Reply:
There is currently a `JmxMeterRegistry` added as well. To avoid changing too much at once, I will keep the `CompositeMeterRegistry`.